### DISCUSSION TITLE
preserve window proportions does not apply to vertical splits

### DISCUSSION THREAD

**Discussion by DylanMartin0930** [nvim-tree/nvim-tree.lua#2116] [Q&A]
### Description

when setting preserve_window_proportions to true, it preserves the sizing of horizontal splits only

### Neovim version

```text
NVIM v0.10.0-dev-1368+gbc66b755f-Homebrew
Build type: Release
LuaJIT 2.1.0-beta3
```

### Operating system and version

Macos Ventura 13.3

### nvim-tree version

fb65be0

### Minimal config

```lua
require("nvim-tree").setup({
view = {
preserve_window_proportions = true,
},
}
```

### Steps to reproduce

1. create vertical split
2. resize it in any direction
3. open and close file tree

### Expected behavior

expect vertical split to stay the same size after toggling file tree on and off

### Actual behavior

toggling file tree on and off causes vertical splits to be resized to be 50/50

**Comment by alex-courtis** [comment]
A little debugging indicates that resizing is not done when opening the tree when `preserve_window_proportions` not set.

It's vim that's doing the resizing, not nvim-tree. You can demonstrate this by opening another vertical split inside a narrow window.

Take a look at `:help wiw` and `:h wmw` to adjust this behaviour.

**Comment by PratyushChaudhary** [comment]
You can fix it by adding this to your config:
`
vim.opt.equalalways = false
`
